### DISCUSSION TITLE
How to use Improved Built-in Fetch Support in server using express.

### DISCUSSION THREAD

**Discussion by starryskadi** [vercel/next.js#13176] [Help]
I would like to know about how to use 'fetch' without installing another package in express server. 
I think it should but can we? Currently I'm using axios package and get rid of it from project if I can use directly 'fetch' from nextjs 9.4.

**Comment by timneutkens** [comment]
The fetch polyfill is introduced when you `require('next')` so if that's the first line of your server it should be fine.
